charter/index.html

```diff
@@ -97,7 +107,14 @@
 
 
     <main>
-        <h1 id="title">Solid Working Group Charter</h1>
+        <h1 id="title">PROPOSED Solid Working Group Charter</h1>
+
+        <div style="border: 1px solid darkred">
+            <p><strong>DRAFT</strong></p>
+            <p>This is a <em>draft</em> text, under development by the community. The goal is to, eventually, submit this draft charter proposal to an official W3C AC review.</p>
```

Comment:
I believe the above line perhaps swings too far in the opposite direction. It's a good idea to mark this as a draft, but this it is the one that went through development and review already and thus it is the one to be reviewed via the official W3C AC process. I believe there is only one item below (the question of the timeline) that needs resolution.

Reply:
this is the boilerplate that we put on all proposed charter, and we remove it just at the moment of submitting it to the AC vote.

Reply:
Sounds good, thanks!